app/models/intervention.rb

```diff
@@ -575,6 +575,22 @@ def working_area(unit = :hectare)
     ActiveSupport::Deprecation.warn 'Intervention#working_area is deprecated. Please use Intervention#working_zone_area instead.'
     working_zone_area(unit)
   end
+  
+  def ratio_for(activity)
```

Comment:
Method name is not very explicit. Change it or add a comment. I prefer largely the first choice.